### PR TITLE
Build app

### PR DESCRIPTION
**Pull request by Theophileaseh**
In this project I,

1. Select the API to use.
2. Build files structure for React app.
3. Prepare routes and navigation in the app.
4. Display a list of items from the API.
5. Create the tests for the application.
6. Style components to match the design provided.
7. Deploy the project and test for final details.
8. Record a video for your project. (https://www.loom.com/share/9cb239054074415c9eb1990a22d10a15)
9. Create a good README and PR description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,68 @@
...
 
-- Major languages
-- Frameworks
-- Technologies used
 
-## Live Demo (if available)
+## Live version
 
-[Live Demo Link](https://livedemo.com)
+For the live version of this project visit the following [Netlify](https://covid-19-updates-hub.netlify.app/) link.
```
Comment (emmanuellekamwa) [inline README.md:21]: - [ ] Your live demo link is not working. Please fix that.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ⛔❌🛑**
Hello @Theophileaseh 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- Your linters are failing. Please fix that.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — CHANGES_REQUESTED** [review]
Hi @Theophileaseh, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] **[REQUIRED]** Kindly set up the linters for react project and ensure that all linter checks are passed. This is needed to approve your project. Follow this [link](https://github.com/microverseinc/linters-config/tree/master/react-redux) to set up the react linter. Also please fix the Netlify checks so they all pass.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @Theophileaseh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**